bambu_spoolman/daemon.py

```diff
@@ -9,9 +10,52 @@
 from bambu_spoolman.broker.automatic_spool_switch import AutomaticSpoolSwitch
 from bambu_spoolman.broker.filament_usage_tracker import FilamentUsageTracker
 from bambu_spoolman.broker.server import run_server
+from bambu_spoolman.spoolman import new_client
+
+
+def wait_for_spoolman(max_retries=30, delay=2):
```

Comment:
Can you elaborate on the use-case for this? I'm assuming its so your spoolman docker container can start up before this app, right?

In that case, this orchestration should be done with healthchecks in the docker compose file and not done by this app.

Reply:
Yes, that was actually my use case :) I think i got carried away while i sucked at setting up the correct healthchecks. Will drop that 🚀